source/blood/src/sectorfx.cpp

```diff
@@ -296,11 +296,11 @@ void DoSectorPanning(void)
                 int px = (pSector->ceilingxpanning<<8)+pXSector->ceilXPanFrac;
                 int py = (pSector->ceilingypanning<<8)+pXSector->ceilYPanFrac;
                 if (pSector->ceilingstat&64)
-                    angle -= 512;
+                    angle += 512;
```

Comment:
These angles based around 2048 = 360 degrees like the [actor ones](https://wiki.eduke32.com/wiki/Ang)? This means add 90 degrees? Just curious.

Reply:
I think it is, and I believe ceiling was meant to have an inverted direction compared to floor panning but that got mixed up along the way. With relative motion the panning matches DOS, so I don't think anyone noticed this.